b3/conf/plugin_adv.ini

```diff
@@ -0,0 +1,40 @@
+[settings]
+# Rate in minutes. If you want a rate faster than once per minutes, then you can add a 's' as a suffix for seconds
+rate: 2
+# You can either set here a text file that will contain one ad per line or fill the <ads> section below
+# ads: c:/somewhere/my_ads.txt
+
+[newsfeed]
+# You can include newsitems in your adds by setting the section below
+# You can add feeditems in the adds like this:
+# @feed   (will pick the next newsitem each time it is included in the rotation,
+#          rotating until 'items' is reached and then start over.)
+# @feed 0 (will pick the latest newsitem available from the feed and add it in the rotation)
+# @feed 1 (will pick the second latest item in line)
+# etc.
+url: http://forum.bigbrotherbot.net/news-2/?type=rss;action=.xml
+items: 5
+pretext: News:
+
+[ads]
+# You can specify advertisements here (one per line)
+# You can use the following keywords in your messages :
+# /spam#rule1 - rule 1 as defined in you admin plugin config file
+# @time - the server time
+# @feed - the next RSS feed item
+# @feed 0 - the lastest RSS feed item
+# @feed 1 - the 2nd lastest RSS feed item
+# @nextmap - the next map in your game server rotation list (if available)
+# @topstats - the top stats from XLRstats (if the plugin is enabled)
+# @admins - connected admins' names. You can change the message format in your admin plugin config
+# @regulars - connected players from the regular grout. You can change the message format in your admin plugin config
+^2Big Brother Bot is watching you... www.BigBrotherBot.net
+@feed
+server watched by @admins
+/spam#rule1
+@time</ad>
```

Comment:
This actually highlighted a limitation: it will be not possible to add the same add multiple time in the ads section since it will be collapsed into a unique key. Don't know how much important this is since I normally do not do that.